Forms/MainForm.cs

```diff
@@ -27,6 +28,10 @@ public partial class MainForm : Form
         public ItemEditForm ItemEditor { get; private set; }
         public StartingItemEditForm StartingItemEditor { get; private set; }
 
+        public readonly string ROOT = Path.GetDirectoryName(System.Reflection.Assembly.GetEntryAssembly().Location) + "\\";
+        public readonly string PresetExt = ".mmrsp";
```

Comment:
This should be a `const` not a `readonly`. I think it's be better to just give it the extensions `.cfg` though.

Reply:
Yeah that's why I made it a variable. I figured we would want to change it and this makes it a lot easier. What is the difference between const and readonly? I was looking for whatever replaced "final" from Java.